BLOGGING.md

```diff
@@ -199,7 +197,7 @@ When you're ready to submit your post for review, issue a Pull Request against t
 
 ## A Note on Dates and Scheduling for Future Publishing
 
-Since the build process is triggered by (and so requires) a commit to `master`, you can either wait for the post's `date` to pass, and _then_ merge it. If a post happens to get merged a future date, the resulting build will exclude the post, requiring a commit of some sort to occur _after_ its `date` in order to trigger a build and get the post published.
+Since the website deployment process is triggered by (and so requires) a commit to pulumi/docs `master`, it isn't possible to schedule a post to be released automatically at a precise date and time. (The `date` frontmatter property has no bearing on whether or when a post will be published; it's used only for sorting and display purposes.) You can, however, influence the timing of the publishing process manually. See the [Merging and Releasing section of the README](README.md#merging-and-releasing) for details.
```

Comment:
```suggestion
It isn't possible to schedule a post to be released automatically at a precise date and time. The `date` frontmatter property is used only for sorting and display purposes. See the [Merging and Releasing section of the README](README.md#merging-and-releasing) to learn how to release at a specific time.
```

i found this sentence kinda difficult to read, and was worried the most important information may get lost in the technical details. but no worries if you feel strongly about it.

Reply:
That's good feedback -- I'll trim things down a bit.